app/controllers/ems_common.rb

```diff
@@ -264,6 +85,33 @@ def show
     end
   end
 
+  def calculate_display_class(display_name, session_display)
+    classes = [Container, ContainerReplicator, ContainerNode, ContainerGroup,
+               ContainerService, ContainerImage, ContainerRoute,
+               ContainerProject, ContainerImageRegistry, AvailabilityZone,
+               CloudTenant, CloudVolume, Flavor, SecurityGroup,]
+    names = classes.collect { |cls| cls.name.underscore.pluralize }
+    if names.include?(display_name)
+      return classes[names.index(display_name)]
+    elsif names.include?(session_display)
+      return classes[names.index(session_display)]
+    end
+    nil
```

Comment:
It is not nitpicking, it is educational as not everyone knows each of these methods to simplify the code.  

Thank you, @DNNX